request.proto

```diff
@@ -166,6 +166,7 @@ message NearestStopPointsRequest {
 message GraphicalIsochroneRequest {
     optional JourneysRequest journeys_request = 1;
     required int32 min_duration               = 2;
+    repeated int32 duration                   = 4;
```

Comment:
That's the API parameter. If that's not clear, maybe we should modify the API parameter. @stifoon ?